src/app/WebValidation/Main.cs

```diff
@@ -412,8 +414,15 @@ public async Task<PerfLog> ExecuteRequest(HttpClient client, string server, Requ
                 requestSummary = RequestSummary;
             }
 
+            string path = request.Path;
+
+            if (!string.IsNullOrWhiteSpace(urlPrefix))
+            {
+                path = urlPrefix + path;
+            }
```

Comment:
Would it be too difficult to update the path in the logs as well? I found it a little confusing to see the original path in the output after passing in a --url-prefix arg. 

![image](https://user-images.githubusercontent.com/3301452/188716017-3b5475bf-ff2a-4bb1-badc-38e90507f790.png)

Reply:
good catch - definitely a bug